aspects/aspects.go

```diff
@@ -56,17 +56,34 @@ func newAccessType(access string) (accessType, error) {
 	return readWrite, fmt.Errorf("expected 'access' to be one of %s but was %q", strutil.Quoted(accessTypeStrings), access)
 }
 
-// NotFoundError represents an error caused by a missing entity.
-type NotFoundError struct {
+// AspectNotFoundError represents a failure to find an aspect.
+type AspectNotFoundError struct {
+	Account string
+	Bundle  string
+	Aspect  string
+}
+
+func (e *AspectNotFoundError) Error() string {
+	return fmt.Sprintf("aspect %s/%s/%s not found", e.Account, e.Bundle, e.Aspect)
+}
+
+func (e *AspectNotFoundError) Is(err error) bool {
+	_, ok := err.(*AspectNotFoundError)
+	return ok
+}
+
+// FieldNotFoundError represents a failure to find a field carrying a value in
+// an aspect databag.
+type FieldNotFoundError struct {
```

Comment:
the spec doesn't use "field" a lot, maybe we will need to have a discussion in one of the next meetings what is the name of aspect entries, entry, field or something else?

Reply:
Sounds good :+1: